code/modules/mob/living/simple_animal/hostile/human/nazi.dm

```diff
@@ -369,6 +369,6 @@
 	say("Eva, auf wiedersehen!")
 	ranged = 0
 	sleep(10)
-	explosion(loc,1,2,3)
-	new/obj/item/weapon/gun/gatling(loc)
 	qdel(src)
+	explosion(loc,1,2,3)
```

Comment:
And now that I think about this, pretty sure this is just going to cause an explosion/spawn in nullspace. Cast `loc` somewhere before the qdel.